docs/metrics.md

```diff
@@ -139,6 +204,36 @@ The logs will be printed to stdout in the standard [mozlog format](https://githu
 }
 ```
 
+And Redshift schema:
+
+```js
+local schema = {
+--   column name                   field type   length  attributes   field name
+    {"timestamp",                  "TIMESTAMP", nil,    "SORTKEY",   "Timestamp"},
```

Comment:
again, copied from the definition: https://mana.mozilla.org/wiki/pages/viewpage.action?pageId=42895640